website/pages/docs/plugins/sources/okta/overview.mdx

```diff
@@ -32,6 +32,14 @@ The following example sets up the Okta plugin, and connects it to a postgresql d
   Token for Okta API access.
   You can set this with an `OKTA_API_TOKEN` environment variable.
 
+- `debug` (`bool`, optional. Default: `false`)
```

Comment:
Could you also add it to the config example?

Reply:
Do we always add every config to the example?